backend/hyperglosae/src/updates/document.js

```diff
@@ -0,0 +1,17 @@
+function(doc, req) {
+  const body = JSON.parse(req.body, (k, v) => (k === 'history') ? undefined : v);
```

Comment:
For your information,  @SanKson7, @lunatiique, and @floporsch911, I have added a "reviver" function parameter, so that someone using a REST client cannot force the history.